lib/Widgets/HyperTextView.vala

```diff
@@ -0,0 +1,324 @@
+/*
+ *  Copyright (C) 2012–2021 elementary, Inc.
+ *
+ *  This program or library is free software; you can redistribute it
+ *  and/or modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 3 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General
+ *  Public License along with this library; if not, write to the
+ *  Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ *  Boston, MA 02110-1301 USA.
+ */
+
+/**
+* This class enables navigatable URLs in Gtk.TextView
+*/
```

Comment:
Sorry, one more minor thing. Since this is a library, we should probably include "since" data. I added a suggestion you can just commit, assuming I guessed the actual release it'll into correctly.

```suggestion
 * @since 6.1.2
 */
```